templates/types/streaming/fastapi/app/api/routers/models.py

```diff
@@ -111,6 +132,18 @@ def get_history_messages(self) -> List[Message]:
     def is_last_message_from_user(self) -> bool:
         return self.messages[-1].role == MessageRole.USER
 
+    def get_chat_document_ids(self) -> List[str]:
+        # Get annotations from the last message has annotations
```

Comment:
@leehuwuj @thucpn as we support a list of doc IDs now, how about we get all doc IDs from all messages (instead of just from the last message)?
the benefit would be that the user could get questions about previous documents answered.
Do you see any downside to that?

Reply:
Yes, that's a great idea. I think there is no issue with that. Let me update it for Python first.

Reply:
Ah, actually i already implemented that, but forgot to remove the comment 😄 . The code's a bit hard to read. Let me clean it up a bit